### PR TITLE
edit locale/en/docs/index.md

### PR DESCRIPTION
**Pull request by Trott**

### CODE REVIEW

File: locale/en/docs/index.md [hunk 1]
```diff
@@ -7,11 +7,17 @@ labels:
 
 # About Docs
 
-It's important for Node.js to provide documentation to its users, but documentation means different things to different people. Here on nodejs.org, you will find three types of documentation: API reference documentation, ES6 features, and frequently asked questions.
+There are three types of documentation available on this website:
+
+* API reference documentation
+* ES6 features
+* frequently asked questions
```
Comment (lpinca) [inline locale/en/docs/index.md]: Capitalize the first word ("Frequently")?

**Comment by lpinca** [comment]
LGTM.

File: locale/en/docs/index.md (hunk 1 above)
Comment (Trott) [inline locale/en/docs/index.md]: Yes, I think that's a good idea. Done and pushed.

**Comment by fhemberger** [comment]
Great work, thank you!

**Pull request merged**